gaia-fast-list.js

```diff
@@ -82,11 +96,25 @@ var GaiaFastListProto = {
       :host {
         display: block;
         height: 100%;
+        overflow: hidden;
       }
 
       .fast-list {
+        position: absolute;
+        left: 0; top: 0;
         height: 100%;
-        position: relative;
+        width: 100%;
+        overflow: hidden;
+      }
+
+      .fast-list.empty {
+        background: repeating-linear-gradient(
+          0deg,
+          var(--background) 0px,
+          var(--background) 59px,
+          var(--border-color) 60px,
+          var(--border-color) 60px);
+        background-position: 0 1px;
```

Comment:
Added some fake list item borders that can be painted when the cache empty (first render).